vpr/src/route/rr_graph_util.h

```diff
@@ -7,7 +7,7 @@ int seg_index_of_cblock(t_rr_type from_rr_type, int to_node);
 
 int seg_index_of_sblock(int from_node, int to_node);
 
-void reorder_rr_graph_nodes(const t_router_opts& router_opts);
+//void reorder_rr_graph_nodes(const t_router_opts& router_opts);
```

Comment:
Delete this line instead of commenting it out.  It is dead code so better not to see it.

Reply:
Removed